low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -250,8 +250,8 @@ def get_cpu_info(self, add_overall_usage=False):
             uid = f"CPU-{cpu_id}"
             cpu_dict = self.get_health_template(uid, is_fru=False)
             online_status = "Online" if cpu_id in cpu_online else "Offline"
-            health_status = "OK" if online_status == "Online" else "NA"
-            usage = "NA" if health_status == "NA" \
+            health_status = "OK" if online_status == "Online" else "FAULT"
```

Comment:
This is wrong, we cant default to Fault health state if health is not available. NA states we cant determine health state of given resource, like here if CPU is offline, same case with inactive service/s ..

Same applies for all other changes here

Reply:
I think Node CLI should have another status as `NA` apart from `FAULT`, `DEGRADED` and `OK`